src/Command/InstallChromeDriverCommand.php

```diff
@@ -0,0 +1,400 @@
+    private const DEFAULT_CHROME_BINARY = 'google-chrome';
+    private const LATEST = 'latest';
+    private const AUTO = 'auto';
+    private const OS_FAMILY = 'os-family';
+    private const WINDOWS = 'Windows';
+    private const BSD = 'BSD';
+    private const DARWIN = 'Darwin';
+    private const SOLARIS = 'Solaris';
+    private const LINUX = 'Linux';
+    private const CHROMEDRIVER_API_URL = 'https://chromedriver.storage.googleapis.com';
+    private const CHROMEDRIVER_API_VERSION_ENDPOINT = self::CHROMEDRIVER_API_URL . '/LATEST_RELEASE';
+    private const BIN_DIRECTORY = __DIR__ . '/../../bin';
+    private const DOWNLOAD_FILE_LOCATION = self::BIN_DIRECTORY . '/chromedriver.zip';
+    private const CHROMEDRIVER_BINARY_LINUX = 'chromedriver_linux64';
+    private const CHROMEDRIVER_BINARY_MAC = 'chromedriver_mac64';
+    private const CHROMEDRIVER_BINARY_WINDOWS = 'chromedriver_win32';
+
+    /**
+     * @var HttpClientInterface
+     */
+    private $httpClient;
+
+    /**
+     * @var Filesystem
+     */
+    private $filesystem;
+
+    /**
+     * @var ZipArchive
+     */
+    private $zip;
+
+    public function __construct(string $name, HttpClientInterface $httpClient, Filesystem $filesystem, ZipArchive $zip)
+    {
+        parent::__construct($name);
+
+        $this->httpClient = $httpClient;
+        $this->filesystem = $filesystem;
+        $this->zip = $zip;
+    }
+
+    protected function configure() : void
+    {
+        $this->setDescription('Installs chrome driver to run panther with Chrome');
+
+        $this->addOption(
+            self::DRIVER_VERSION,
+            null,
+            InputOption::VALUE_OPTIONAL,
+            sprintf('The chrome driver version to install (x.x.x.x|%s|%s)', self::LATEST, self::AUTO),
+            self::AUTO
+        );
+
+        $this->addOption(
+            self::CHROME_BINARY,
+            null,
+            InputOption::VALUE_OPTIONAL,
+            sprintf(
+                'The chrome binary used to determine the correct chrome driver version when --%s=%s',
+                self::DRIVER_VERSION,
+                self::AUTO
+            ),
+            self::DEFAULT_CHROME_BINARY
+        );
+
+        $this->addOption(
+            self::OS_FAMILY,
+            null,
+            InputOption::VALUE_OPTIONAL,
+            sprintf(
+                'The OS family used for installing the correct chrome driver binary (%s|%s|%s|%s|%s)',
+                self::WINDOWS,
+                self::BSD,
+                self::DARWIN,
+                self::SOLARIS,
+                self::LINUX
+            ),
+            PHP_OS_FAMILY
+        );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output) : int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $io->note('This command is experimental. Use at your own discretion.');
+
+        $driverVersion = $input->getOption(self::DRIVER_VERSION);
+        if ($driverVersion === self::AUTO) {
```

Comment:
You should use a `switch` here (nitpicking but it's the appropriate construct and it is a bit faster, even if it will not be noticeable here).